LightingApi/src/api.py

```diff
@@ -102,6 +103,144 @@ def bulb_2() -> Response:
+            ServiceUris.BULB_SERVICE + "/request/bulb2", json=bulb_2_rose_request
+        )
+        insert_lighting_request(device_name="ledstrip", request=request)
+        insert_lighting_request(device_name="bulb1", request=request)
+        insert_lighting_request(device_name="bulb2", request=request)
+        return "Success", 200
+    except requests.HTTPError as e:
+        return str(e), 500
+
+
+@app.route("/lighting/scene/rainbow", methods=["POST"])
+def rainbow() -> Response:
+    led_strip_rainbow_request = dict(operation="rainbow")
+    bulb_1_rainbow_request = dict(operation="rainbow")
+    bulb_2_rainbow_request = dict(operation="rainbow")
+    try:
+        requests.post(
+            ServiceUris.LED_STRIP_SERVICE + "/request", json=led_strip_rainbow_request
+        )
+        requests.post(
+            ServiceUris.BULB_SERVICE + "/request/bulb1", json=bulb_1_rainbow_request
+        )
+        requests.post(
+            ServiceUris.BULB_SERVICE + "/request/bulb2", json=bulb_2_rainbow_request
+        )
+        insert_lighting_request(device_name="ledstrip", request=request)
+        insert_lighting_request(device_name="bulb1", request=request)
+        insert_lighting_request(device_name="bulb2", request=request)
+        return "Success", 200
+    except requests.HTTPError as e:
+        return str(e), 500
+
+
+@app.route("/lighting/scene/candy", methods=["POST"])
+def candy() -> Response:
+    led_strip_candy_request = dict(operation="hsv", h=260, s=63, v=96)
+    bulb_1_candy_request = dict(operation="hsv", h=301, s=55, v=98)
+    bulb_2_candy_request = dict(operation="hsv", h=270, s=69, v=87)
+    try:
+        requests.post(
+            ServiceUris.LED_STRIP_SERVICE + "/request", json=led_strip_candy_request
+        )
+        requests.post(
+            ServiceUris.BULB_SERVICE + "/request/bulb1", json=bulb_1_candy_request
+        )
+        requests.post(
+            ServiceUris.BULB_SERVICE + "/request/bulb2", json=bulb_2_candy_request
+        )
+        insert_lighting_request(device_name="ledstrip", request=request)
+        insert_lighting_request(device_name="bulb1", request=request)
+        insert_lighting_request(device_name="bulb2", request=request)
+        return "Success", 200
+    except requests.HTTPError as e:
+        return str(e), 500
+
+
+@app.route("/lighting/scene/peachy", methods=["POST"])
+def peachy() -> Response:
+    led_strip_peachy_request = dict(operation="hsv", h=17, s=90, v=100)
+    bulb_1_peachy_request = dict(operation="hsv", h=27, s=59, v=100)
+    bulb_2_peachy_request = dict(operation="hsv", h=54, s=70, v=94)
+    try:
+        requests.post(
+            ServiceUris.LED_STRIP_SERVICE + "/request", json=led_strip_peachy_request
+        )
+        requests.post(
+            ServiceUris.BULB_SERVICE + "/request/bulb1", json=bulb_1_peachy_request
+        )
+        requests.post(
+            ServiceUris.BULB_SERVICE + "/request/bulb2", json=bulb_2_peachy_request
+        )
```

Comment:
For this function and all others, put a newline between the last post and the fist insert lighting request. When writing clean code it's nice sometimes to seperate logical blocks by spaces, so just adding a quick space lets us see that it's a different piece of logic (the posts versus database insertions).